cli/command/telemetry_docker.go

```diff
@@ -1,3 +1,6 @@
+// FIXME(jsternberg): remove once we are a module; the go:build directive prevents go from downgrading language version to go1.16:
+//go:build go1.19
```

Comment:
is there a reason for 1.19 over 1.18?

Reply:
This seems to be the version that was chosen by @thaJeztah so I just copied it.

https://github.com/docker/cli/commit/70216b662dc440faaebab531deb35f0f0c633d17

This was the reasoning:

> It's currently using go1.19 as version to match the version
in our "vendor.mod", but we can consider being more permissive ("any" requires
go1.18 or up), or more "optimistic" (force go1.21, which is the version we
currently use to build).